src/engraving/libmscore/chordlist.cpp

```diff
@@ -1432,6 +1434,799 @@ qreal ChordList::position(const QStringList& names, ChordTokenClass ctc) const
     }
 }
 
+//---------------------------------------------------------
+//   findModifierStartIndices
+//---------------------------------------------------------
+void ParsedChord::findModifierStackIndices()
+{
+    QStringList alterationsList = { "b", "bb", "#", "##", "natural" };
+    QStringList addOmitList = { "omit", "no", "add" };
```

Comment:
Better to declare those lists as static const at the top of the file and use them wherever needed